sources/packages/forms/src/form-definitions/uploadstudentdocuments.json

```diff
@@ -1,12 +1,13 @@
 {
   "title": "Upload Student Documents",
-  "display": "form",
-  "type": "form",
   "name": "uploadStudentDocuments",
   "path": "uploadstudentdocuments",
+  "type": "form",
+  "display": "form",
   "tags": [
     "common"
   ],
+  "owner": "63bf288c3e8304f36f5f1717",
```

Comment:
We can get rid of the owner tag for this and every form.io definition that we updated.

Reply:
I will create a Pr after this to remove all other "owner"